resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -444,6 +444,17 @@ private[spark] object Config extends Logging {
       .stringConf
       .createWithDefault("direct")
 
+  val KUBERNETES_EXECUTOR_POD_SNAPSHOT_SOURCES =
+    ConfigBuilder("spark.kubernetes.executor.pod.snapshotSources")
+      .doc("Class names of pod snapshot sources implementing " +
+        "ExecutorPodsCustomSnapshotSource. This is a developer API. Comma separated. " +
+        "If not specified, the default snapshot sources, ExecutorPodsWatchSnapshotSource" +
```

Comment:
nit. We need an extra space, `ExecutorPodsWatchSnapshotSource"` -> `ExecutorPodsWatchSnapshotSource "`